### ISSUE TITLE
Find a better terminology than "asynchronous" or "deferred" credential issuance

### ISSUE THREAD

**Issue by Sakurann** [tlodderstedt/openid-connect-4-credential-issuance#3]
Find a better terminology than "asynchronous" or "deferred" credential issuance for a use-case, when credential is issued not right after the request, but following certain offline business processes. 

Looks like for some in SSI, 'asynchronous' might mean issuer sending the issued credential directly to the user, without user having to go to the issuer with a receipt token to obtain a credential once it is ready.

**Comment by tlodderstedt** [comment]
PR #8

**Comment by selfissued** [comment]
I personally think "asynchronous" is the clearest term, and our use of it matches what it means in other usages in Computer Science.

**Comment by Sakurann** [comment]
I have done some asking around and it does sound like for some people in SSI, "asynchronous" does mean the Issuer pushing the issued credential directly to the user, which is not what I mean. I suggest we go with "deferred" and might revisit later once we get more eyes on this work.

**Comment by Sakurann** [comment]
PR #8 merged
